substra/cli/printers.py

```diff
@@ -266,6 +275,7 @@ class ComputePlanPrinter(AssetPrinter):
         ProgressField('Progress', 'doneCount', 'tupleCount'),
         Field('Status', 'status'),
         Field('Tag', 'tag'),
+        Field('Metadata', 'metadata'),
```

Comment:
It looks like there are a few metadata fields in the "list_fields" sections, they shouldn't be here as they will break the output (they are too long to be printed in a table).